lms/services/user.py

```diff
@@ -17,10 +17,17 @@ class UserService:
     At the moment this is purely used for recording/reporting purposes.
     """
 
-    def __init__(self, application_instance_service, db, h_authority: str):
+    def __init__(self, application_instance_service, db, h_authority: str, request):
         self._application_instance_service = application_instance_service
         self._db = db
         self._h_authority = h_authority
+        self._request = request
+
+    def get_current(self):
```

Comment:
Matches the naming and functionality of `application_instance_service.get_current`.

As the result would be available directly on request we won't see much usage of this but I think is nice to have matching APIs in the services.